qelem/lib/screens/splash_screen.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+class Splash extends StatefulWidget {
```

Comment:
Could you name it "SplashScreen" instead of splash?

Reply:
will do